lib/controller_command_proc.rb

```diff
@@ -7,12 +7,13 @@ def initialize
     @bot_dbaccess        = $bot_dbaccess
   end
 
-  def whitelist
-    ['move_absolute','move_relative','home_x','home_y','home_z','calibrate_x','calibrate_y','calibrate_z','dose_water','set_speed','pin_write','pin_read','pin_mode','pin_pulse','servo_move']
-  end
+  WHITELIST = ['move_absolute','move_relative','home_x','home_y','home_z',
+               'calibrate_x','calibrate_y','calibrate_z','dose_water',
+               'set_speed','pin_write','pin_read','pin_mode','pin_pulse',
+               'servo_move']
 
   def check_whitelist(function)
-    raise "UNAUTHORIZED" unless whitelist.include?(function.downcase)
+    raise "UNAUTHORIZED" unless WHITELIST.include?(function.downcase)
```

Comment:
Converted this to a class level constant so I could re-use it in the websocket porition of the app